shared/castable-video/index.mjs

```diff
@@ -0,0 +1,3 @@
+// this is a ecmascript module entry for castable-video to remove the need to build for this package.
+// Since the castable-video will automatically bind to globalThis, no export is necessary.
+import './castable-video.js';
```

Comment:
nit (non-blocking): fine for now, but we should probably export this longer term.